src/theme/book.js

```diff
@@ -6,12 +6,19 @@ window.onunload = function () { };
 // Global variable, shared between modules
 function playpen_text(playpen) {
     let code_block = playpen.querySelector("code");
+    let regex = /^\$(?: )/;
```

Comment:
I would suggest a more explicit variable name for this regex.
```suggestion
    let regex_leading_dollar_space = /^\$(?: )/;
```